.gitignore

```diff
@@ -5,6 +5,8 @@ yarn-error.log
 .vscode
 bin
 .DS_Store
+package-lock.json
+yarn.lock
```

Comment:
```suggestion
```
These files should be tracked. Otherwise, the expected packages might not be installed and make dependency conflict.

Reply:
Got it.